src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -5114,6 +5114,8 @@ static SimdIntrinsic packedsimd_methods [] = {
 	{SN_Dot, OP_XOP_X_X_X, INTRINS_WASM_DOT},
 	{SN_ExtractLane},
 	{SN_Floor, OP_XOP_OVR_X_X, INTRINS_SIMD_FLOOR},
+	{SN_LoadScalarVector128},
+	{SN_LoadVector128, OP_LOADX_MEMBASE},
 	{SN_Max, OP_XBINOP, OP_IMIN, OP_XBINOP, OP_IMIN_UN, OP_XBINOP, OP_FMIN},
```

Comment:
This probably needs more processing.

Reply:
Could you please go into details? Not sure what needs more processing.

Reply:
Load opcodes have an inst_offset field which might need to be set, although in this case, its probably 0.